coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 94.81
+    "covered_percent": 94.79
```

Comment:
It'd be good to take a look at this and see why coverage percentage is declining.